K8s/install-all.sh

```diff
@@ -16,3 +16,6 @@ kubectl apply -f mongodb/mongodb-exporter-servicemonitor.yaml
 #helm install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx -f ingress/values.yaml -n robot-shop
 
 kubectl apply -f ingress/ingress.yaml -n robot-shop
+
+# Enable Rabbit MQ Native Prometheus Exporter
```

Comment:
We can remove the `rabbitmq-exporter` helm install call above as well as the `rabbitmq-values.yaml` file